feature-libs/my-account/organization/core/services/organization-page-meta.resolver.ts

```diff
@@ -22,35 +30,79 @@ import { map } from 'rxjs/operators';
   providedIn: 'root',
 })
 export class OrganizationPageMetaResolver
-  extends ContentPageMetaResolver
-  implements PageBreadcrumbResolver {
+  extends PageMetaResolver
+  implements PageBreadcrumbResolver, PageTitleResolver {
   pageTemplate = 'CompanyPageTemplate';
-  protected ORGANIZATION_TRANSLATION_KEY = 'organization.breadcrumb';
+  pageType = PageType.CONTENT_PAGE;
+
+  /**
+   * Translation key for the breadcrumb of Organization home page
+   */
+  protected readonly ORGANIZATION_TRANSLATION_KEY = 'organization.breadcrumb';
+
+  /**
+   * Semantic route name of the Organization home page
```

Comment:
Can we add a note here, that the `ORGANIZATION_SEMANTIC_ROUTE` is used toe recognise the organisation landing page to avoid showing the breadcrumb for this page? It does not contribute to the router logic, it's only to prevent the superfluous breadcrumb. 

nitpick: I think we agreed on ending single sentences in docs with a dot.